lib/cc/cli/analyze.rb

```diff
@@ -83,7 +73,7 @@ def engines
               engine_name,
               entry,
               path,
-              engine_config(engine_name),
+              config.merge!(exclude_paths: exclude_paths).to_json,
```

Comment:
Does this actually mutate the `CC::Yaml` object we pass around? Would it be better to use `merge` here?